### ISSUE TITLE
Error when Finetuning with local dataset

### ISSUE THREAD

**Issue by Mewral** [mosaicml/llm-foundry#213]
Hi, I'm fine-tuning mpt-7b with my local dataset. Dataset processor seems fine but getting error while training.
I get this warning says 
```python
UserWarning: Truncating TARGET sequence of length=7704 to length=2040, so context+target fit max_seq_len=2048. If truncation is a problem, consider increasing max_seq_len.
```
then get an error:
```python
Unable to create tensor, you should probably activate truncation and/or padding with 'padding=True' 'truncation=True' to have batched tensors with the same length. Perhaps your features (`input_ids` in this case) have excessive nesting (inputs type `list` where type `int` is expected)
```

How do I activate truncation or padding in my yaml? Can't find any information in examples

**Comment by palash04** [comment]
@Mewral , Where did you add 'padding=True' information? I am getting the same error.
